Share Extension/ShareViewController.swift

```diff
@@ -1,22 +1,59 @@
 import UIKit
+import UniformTypeIdentifiers
 import Social
 
-class ShareViewController: SLComposeServiceViewController {
+class ShareViewController: UIViewController {
 
-    override func isContentValid() -> Bool {
-        // Do validation of contentText and/or NSExtensionContext attachments here
-        return true
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        let content = extensionContext?.inputItems.first as? NSExtensionItem
+        guard let attachment = content?.attachments?.first as? NSItemProvider else {
+            close()
+            return
+        }
+
+        if attachment.hasItemConformingToTypeIdentifier(UTType.data.identifier) {
+
+            attachment.loadItem(forTypeIdentifier: UTType.data.identifier, options: nil) { [weak self] data, error in
+                guard let url = data as? URL else {
+                    self?.close()
+                    return
+                }
+
+                guard let opmlData = try? Data(contentsOf: url) else {
+                    self?.close()
+                    return
+                }
+
+                self?.close()
+
+                self?.redirectToHostApp(opmlData.base64EncodedString())
```

Comment:
My original approach was to copy the file to the shared directory but our current provisioning profile is not ready for the app group.

I decided to go ahead with the Base 64 approach given it works and [it's somewhat safe](https://stackoverflow.com/a/58869478).

Once the provisioning profile is done I can revert to this commit: https://github.com/Automattic/pocket-casts-ios/pull/989/commits/660f69a17208b63cd755ae757addead291b9fafa

Reply:
Trying to understand what this code does and why we need it and I think I kinda understand. I think some in code documentation can be helpful. 
`if attachment.hasItemConformingToTypeIdentifier(UTType.data.identifier)` - when will it not conform?

Reply:
> I think some in code documentation can be helpful.

I'll add!

> when will it not conform?

This should be hit only, and if only, the shared file is an `OPML` file (which has the `.data` type). We can be sure of that because the share extension is activated only for `OPML` as defined here: https://github.com/Automattic/pocket-casts-ios/pull/989/files#diff-014aaa0834a60bcc7c5f4da354f564222d6bdcc7ac2da030d5c1e9189bccb00cR11-R22

I'll add documentation to make that clear. :)

Reply:
@yaelirub added documentation in 35f0086952a10eae7064cde5b02a1dcfcf388f62